app/controllers/dashboards_controller.rb

```diff
@@ -35,4 +36,8 @@ def defence_requests_scoped_by_policy
   def policy_context
     @_policy_context ||= PolicyContext.new(DefenceRequest, current_user)
   end
+
+  def client
+    @_client ||= ServiceRegistry.service(:auth_api_client).new(session[:user_token])
```

Comment:
Seems we're not using a @_ prefix convention consistently across controllers. Perhaps we could rename these to `@client` and `@policy_context`?